web/wp-content/themes/dctx-v2/inc/template-tags.php

```diff
@@ -0,0 +1,540 @@
+ *
+ * @author WDS
+ * @return string
+ */
+function dctx_get_the_excerpt( $args = array() ) {
+
+	// Set defaults.
+	$defaults = array(
+		'length' => 20,
+		'more'   => '...',
+		'post'   => '',
+	);
+
+	// Parse args.
+	$args = wp_parse_args( $args, $defaults );
+
+	// Trim the excerpt.
+	return wp_trim_words( get_the_excerpt( $args['post'] ), absint( $args['length'] ), esc_html( $args['more'] ) );
+}
+
+/**
+ * Echo the copyright text saved in the Customizer.
+ *
+ * @author WDS
+ * @return bool
+ */
+function dctx_display_copyright_text() {
+
+	// Grab our customizer settings.
+	$copyright_text = get_theme_mod( 'dctx_copyright_text' );
+
+	// Stop if there's nothing to display.
+	if ( ! $copyright_text ) {
+		return false;
+	}
+
+	echo dctx_get_the_content( do_shortcode( $copyright_text ) ); // phpcs: xss ok.
+}
+
+/**
+ * Get the Twitter social sharing URL for the current page.
+ *
+ * @author WDS
+ * @return string
+ */
+function dctx_get_twitter_share_url() {
+	return add_query_arg(
+		array(
+			'text' => rawurlencode( html_entity_decode( get_the_title() ) ),
+			'url'  => rawurlencode( get_the_permalink() ),
+		),
+		'https://twitter.com/share'
+	);
+}
+
+/**
+ * Get the Facebook social sharing URL for the current page.
+ *
+ * @author WDS
+ * @return string
+ */
+function dctx_get_facebook_share_url() {
+	return add_query_arg( 'u', rawurlencode( get_the_permalink() ), 'https://www.facebook.com/sharer/sharer.php' );
+}
+
+/**
+ * Get the LinkedIn social sharing URL for the current page.
+ *
+ * @author WDS
+ * @return string
+ */
+function dctx_get_linkedin_share_url() {
+	return add_query_arg(
+		array(
+			'title' => rawurlencode( html_entity_decode( get_the_title() ) ),
+			'url'   => rawurlencode( get_the_permalink() ),
+		),
+		'https://www.linkedin.com/shareArticle'
+	);
+}
+
+/**
+ * Display the social links saved in the customizer.
+ *
+ * @author WDS
+ */
+function dctx_display_social_network_links() {
+
+	// Create an array of our social links for ease of setup.
+	// Change the order of the networks in this array to change the output order.
+	$social_networks = array( 'facebook', 'instagram', 'linkedin', 'twitter' );
+
+	?>
+	<ul class="social-icons">
+		<?php
+		// Loop through our network array.
+		foreach ( $social_networks as $network ) :
+
+			// Look for the social network's URL.
+			$network_url = get_theme_mod( 'dctx_' . $network . '_link' );
+
+			// Only display the list item if a URL is set.
+			if ( ! empty( $network_url ) ) :
+			?>
+				<li class="social-icon <?php echo esc_attr( $network ); ?>">
+					<a href="<?php echo esc_url( $network_url ); ?>">
+						<?php
+						dctx_display_svg(
+							array(
+								'icon' => $network . '-square',
+							)
+						);
+						?>
+						<span class="screen-reader-text">
+						<?php
+							echo /* translators: the social network name */ sprintf( esc_html( 'Link to %s', 'dctx' ), ucwords( esc_html( $network ) ) ); // WPCS: XSS OK.
```

Comment:
:warning: **Warning**: Using the WPCS native whitelist comments is deprecated. Please use the PHPCS native &quot;phpcs:ignore Standard.Category.SniffName.ErrorCode&quot; annotations instead. Found: // WPCS: XSS OK.\n (*WordPress.Security.EscapeOutput.DeprecatedWhitelistCommentFound*).